.github/workflows/build-latest-nightly.yml

```diff
@@ -0,0 +1,28 @@
+name: Build latest nightly
+on:
+  schedule:
+    - cron: "0 0 * * *" # daily at midnight
+
+jobs:
+  test:
+    name: asdf-nim plugin test
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # Test installation of official Linux binaries from nim-lang.org
+          - os: ubuntu-latest
+            nim-version: latest
+
+          # 2020-01-01 - macOS builds across repos on Github Actions are timing out, so disabling for now
+          # - os: macos-latest
+          #   nim-version: latest
+
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Test nimble ${{ matrix.nim-version }}
+        uses: asdf-vm/actions/plugin-test@v1
+        with:
+          command: nim -v
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
You can remove this line.

Reply:
Removed in 1a414c1d87d9fed2128ad1fd9ccee370c1d0c91f

Reply:
It’s same to `.github/workflows/build.yml`

Reply:
Removed in 1e9e7ed3819a4ed39aa5fae3595b91a6b9d8767f

Reply:
Looks good!